source/AnimatedEntity.cpp

```diff
@@ -0,0 +1,54 @@
+#include "AnimatedEntity.hpp"
+
+AnimatedEntity::AnimatedEntity(Renderer & renderer, std::string const & mesh)
+  : AnimatedEntity(renderer, mesh, renderer.getSceneManager().getRootSceneNode()->createChildSceneNode())
+{}
+
+AnimatedEntity::AnimatedEntity(Renderer & renderer, std::string const & mesh, Ogre::SceneNode * parent)
+  : entity(renderer, mesh, parent)
+{}
+
```

Comment:
If the constructors only do forwarding, consider doing perfect forwarding instead.